core/src/main/java/org/elasticsearch/rest/action/bulk/RestBulkAction.java

```diff
@@ -93,6 +93,9 @@ public void handleRequest(final RestRequest request, final RestChannel channel,
             public RestResponse buildResponse(BulkResponse response, XContentBuilder builder) throws Exception {
                 builder.startObject();
                 builder.field(Fields.TOOK, response.getTookInMillis());
+                if (response.getIngestTookInMillis() != 0) {
```

Comment:
I think it's possible the ingest phase will take genuinely take 0 millis (i.e. <1ms)? in that case we want to report. I would suggest using a negative value to indicate "ingest never run" and suppress rendering in that case alone.